extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/functionTypeExtractors.scala

```diff
@@ -33,6 +36,18 @@ object FunctionTypeExtractor {
   val functionTypeExtractors: Map[String, FunctionTypeExtractor] = {
     loadExtractorsToMap[FunctionTypeExtractor]
   }
+
+  def getFunctionType(fi: FunctionIdentifier, catalog: SessionCatalog): FunctionType = {
+    if (catalog.isTemporaryFunction(fi)) {
+      TEMP
+    } else if (catalog.isPersistentFunction(fi)) {
+      PERMANENT
+    } else if (catalog.isRegisteredFunction(fi)) {
+      SYSTEM
+    } else {
+      TEMP
```

Comment:
For repeated TEMP in branches, consider use match-case to generify them.

Reply:
> For repeated TEMP in branches, consider use match-case to generify them.

ok